submissions/xT0PERx/js-dom/code.js

```diff
@@ -1,65 +1,71 @@
+      '<img class="img-content" src=https://i1.wp.com/autogeek.com.ua/wp-content/uploads/2017/01/3f39410d0e6da7cacc8b6f4260bfd6d4-387x480.jpg?resize=387%2C480><p> Tesla (ранее Tesla Motors) — американская компания, производитель электромобилей и (через свой филиал SolarCity) решений для хранения электрической энергии.</p><p>Tesla разворачивает сеть «Суперзарядок» (англ. Supercharger) — станций для зарядки электромобилей, разработанных для того, чтобы на автомобилях Tesla можно было совершать длительные поездки. </p>',
   },
 ];
 
-document.querySelector(".menu-list").addEventListener("click", function (e) {
-  if (e.target && e.target.nodeName === "LI") {
+document.querySelector('.menu-list').addEventListener('click', function (e) {
```

Comment:
You can use [Destructuring_assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here for unpacking field `target` from event-object passed as function parameter.

So you can do `function ({ target }) { ...` and you don't need to write `e.target`

Reply:
Not fixed yet)